integrationtests/test_notifications.py

```diff
@@ -54,7 +54,15 @@ def got_notification_email(_unused):
             if not message:
                 return False
             # Get payload at index 1: the text/html payload
-            body = message.get_payload()[1].get_payload()
+            #
+            # There is a chance that an old signup message will be fetched
+            # before the new notification is written to disk;
+            # the signup message will have a `str` at index 1, resulting in
+            # an `AttributeError` for the second `get_payload`
```

Comment:
You figured it out! Thank you! (I'd seen this error under certain conditions and never figured it out.)